### PR TITLE
MTDSA-6131 RAML for Amend a Foreign Property Income & Expenditure Per…

### PR DESCRIPTION
**Pull request by HarryBaxter**
…iod Summary

### CODE REVIEW

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "method":"PUT",
+      "rel":"amend-property-period-summary"
+    },
+    {
+      "href":"/Individuals/business/property/{nino}/{businessId}/period",
+      "method":"GET",
+      "rel":"self"
+    }
+  ]
+}
```
Comment (Mbrookesy) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:19]: I feel the url values for nino, businessId and submissionId should have actual values in them to show a proper response example.

File: resources/public/api/conf/1.0/examples/amend_foreign_property_request.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    "expenditure": {
+      "premisesRunningCosts": 4567.98,
+      "repairsAndMaintenance": 98765.67,
+      "financialCosts": 4566.95,
+      "professionalFees": 23.65,
+      "costsOfServices": 4567.77,
+      "travelCosts": 456.77,
+      "other": 567.67,
+      "consolidatedExpenses": 456.98
```
Comment (Mbrookesy) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_request.json:15]: Due to the `RULE_BOTH_EXPENSES_SUBMITTED` error I don't believe this request is a valid one

File: resources/public/api/conf/1.0/examples/amend_foreign_property_request.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "premisesRunningCosts": 5635.43,
+        "repairsAndMaintenance": 3456.65,
+        "financialCosts": 34532.21,
+        "professionalFees": 32465.32,
+        "costsOfServices": 2567.21,
+        "travelCosts": 2345.76,
+        "residentialFinancialCost": 21235.22,
+        "broughtFwdResidentialFinancialCost": 12556.00,
+        "other": 2425.11,
+        "consolidatedExpenses": 352.66
```
Comment (Mbrookesy) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_request.json:42]: Ditto

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "links":[
+    {
+      "href":"/Individuals/business/property/TC663795B/XAIS12345678910/period/4557ecb5-fd32-48cc-81f5-e6acd1099f3c",
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:4]: remove capital `I`

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      "method":"GET",
+      "rel":"retrieve-property-period-summary"
+    },
+    {
+      "href":"/Individuals/business/property/TC663795B/XAIS12345678910/period/4557ecb5-fd32-48cc-81f5-e6acd1099f3c",
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:9]: remove capital `I`

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+      "method":"PUT",
+      "rel":"amend-property-period-summary"
+    },
+    {
+      "href":"/Individuals/business/property/TC663795B/XAIS12345678910/period",
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:14]: remove capital `I`

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      "method":"GET",
+      "rel":"self"
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:16]: this rel should be `list-property-period-summaries`

File: resources/public/api/conf/1.0/examples/amend_foreign_property_response.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
+{
+  "links":[
+    {
+      "href":"/Individuals/business/property/TC663795B/XAIS12345678910/period/4557ecb5-fd32-48cc-81f5-e6acd1099f3c",
+      "method":"GET",
+      "rel":"retrieve-property-period-summary"
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/amend_foreign_property_response.json:6]: this rel should be `self`

File: resources/public/api/conf/1.0/schemas/amend_foreign_property_response.json [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+            "enum": [
+              "GET",
+              "PUT"
+            ]
+          },
+          "rel": {
+            "description": "A label for the endpoint, which describes how it is related to the current resource. The rel will be self where the action is retrieval of the same resource",
+            "type": "string",
+            "enum": [
+              "retrieve-property-period-summary",
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/schemas/amend_foreign_property_response.json:31]: change to `list-property-period-summaries`

**Pull request merged**